boututils/showdata.py

```diff
@@ -688,7 +688,7 @@ def init():
                      print('Save failed: Check ffmpeg path')
                      raise
         elif movietype == 'gif':
-            anim.save(movie+'.gif',writer = 'imagemagick', fps=fps, dpi=dpi)
+            anim.save(movie+,writer = 'imagemagick', fps=fps, dpi=dpi)
```

Comment:
Is there a spurious `+` in the line? Should it rather be

```suggestion
            anim.save(movie, writer = 'imagemagick', fps=fps, dpi=dpi)
```